src/components/contactBottomHeaderArea.js

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from "react"
+import "./contactBottomHeaderArea.scss"
+
+
+class ContactBottomHeaderArea extends Component {
+    constructor(props) {
+        super(props);
+    
+        this.state = {
+          firstName: "",
+          lastName: "",
+          email: "",
+          subject: "",
+          message: "",
+        };
+      }
+
+    encode = (data) => {
+        return Object.keys(data)
+            .map(key => encodeURIComponent(key) + "=" + encodeURIComponent(data[key]))
+            .join("&")
+      }
+    
+    handleSubmit = (event) => {
+      event.preventDefault()
+      fetch("/", {
+        method: "POST",
+        headers: { "Content-Type": "application/x-www-form-urlencoded" },
+        body: this.encode({
+          "form-name": "contact-form",
+          ...this.state
+        })
+      }).then(() => alert("Success!"))
+        .catch(error => alert(error));
```

Comment:
We need to update these still to show a message below the submit button on success or error, rather than using an alert message.